src/components/App/DisplayPuzzleInstances/DisplayPuzzleInstances.module.scss

```diff
@@ -0,0 +1,88 @@
+@import '../../../styles/abstracts/variables';
+@import '../../../styles/abstracts/_mixins/text.scss';
+
+.puzzleInstancesWrapper {
+  display: flex;
+  flex-direction: column;
+
+  .puzzleInstancesHeader {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 2rem;
+
+    .puzzleInstancesHeaderSelectWrapper {
+      @include font-button;
+      color: $black;
+      display: inline-block;
+      overflow: hidden;
+      border: 2px solid #4b4b4b;
```

Comment:
Is this a non-variable colour I see? 👀

Reply:
Oopsie

Reply:
Oh yeah I copied this from `Input.module.scss`

Reply:
I'll change that